### PR TITLE
Add contact page

### LINKED ISSUE #1761: Create contact page for federalist.18f.gov (dev)
#### User Story

As a Federalist user or potential user, I need a way to know how to contact the Federalist team and expect that be easily found.

#### Acceptance Criteria

- [ ] Page exists at `/contact`
- [ ] Page is linked from homepage

#### Implementation steps

- [x] Create a page framework for this content 
- [x] Page is populated with the content from #1795 
- [ ] Page is pushed to and displaying on staging.

#### Level of effort - low

### PR DESCRIPTION
**Pull request by line47**
this closes #1761

### CODE REVIEW

**Comment by line47** [comment]
I'm not sure why the `total-coverage` went down here ?

File: api/routers/main.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ const csrfProtection = require('../policies/csrfProtection');
 router.get('/', MainController.home);
 router.get('/case-studies/', MainController.examples);
 router.get('/content/examples', (req, res) => res.redirect('/case-studies'));
+router.get('/contact/', MainController.contact);
+router.get('/content/contact', (req, res) => res.redirect('/contact'));
```
Comment (wslack) [inline api/routers/main.js]: I don't think we need this; the other line was to keep old links compatible with the new site structure.

File: views/content/contact.njk [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{% extends "base.njk" %}
+
+{% block main_content %}
+  <div class="usa-grid info-block usa-content">
+    <h1>Contact</h1>
+    <h2>Want to buy Federalist?</h2>
+    <p>If you’re interested in using Federalist,
+      <a href="mailto:cloud-gov-inquiries@gsa.gov?body=What%27s%20your%20name%3F%0A%0AWhat%20agency%20or%20office%20do%20you%20work%20for%3F%0A%0AWhat%27s%20your%20job%20title%20or%20role%3F%0A%0ATell%20us%20a%20little%20about%20your%20project%20or%20your%20questions%20about%20cloud.gov:%0A%0AIf%20you%27d%20like%20us%20to%20call%20you%2C%20what%27s%20your%20phone%20number%20and%20when%20might%20be%20a%20good%20time%3F%0A">email the Federalist team</a> with your name, agency or office, and information about the website you want to build or migrate. We’ll schedule a call to answer your questions and help you get started.</p>
```
Comment (wslack) [inline views/content/contact.njk]: Needs new content

File: views/home.njk [hunk 3]
```diff
@@ -233,7 +233,7 @@
                 Want to talk about what Federalist can do for your agency? Questions about our ATO? Want to try it out for yourself?
               </p>
               <p>
-                <a class="usa-button" href="https://docs.google.com/forms/d/1iB8aW7c9r1QH3s8XElQCrnXRGjAiPUYpWG1CMeEqGIo/viewform">Get in touch</a>
+                <a class="usa-button" href="/contact/">Get in touch</a>
```
Comment (wslack) [inline views/home.njk]: Don't need this change given where this is on page - right @annahsebok?

**Comment by wslack** [comment]
@line47  - I dropped in a few comments - do you have a sec to take a look? cc @annahsebok since on

**Comment by wslack** [comment]
I think the test coverage drop is from not having something like https://github.com/18F/federalist/blob/cde92087ea3a850dd246ef27445cc91a3997a56c/test/api/requests/main.test.js#L160

File: views/content/contact.njk (hunk 2 above)
Comment (line47) [reply views/content/contact.njk]: Content for the mailto: link or content for this entire page.
I pulled the latest from https://docs.google.com/document/d/1PYVw2IgRCuatdtpn1_0nKBL8JBV4i5VbyARnFJuzk3o/edit#heading=h.92izs1cnqim7

**Comment by line47** [comment]
Yeah we can one 👍

File: views/home.njk (hunk 3 above)
Comment (wslack) [reply views/home.njk]: @line47 mind removing this change for now? I can put it back in later if we need.

Comment (line47) [reply views/home.njk]: No problem

**Comment by wslack** [comment]
@amirbey to check when back for why codeclimate is stuck

**Comment by wslack** [comment]
aha! unstuck

File: views/home.njk (hunk 3 above)
Comment (wslack) [reply views/home.njk]: as soon as this is changed back, I think this can be merged.

**Comment by amirbey** [comment]
@wslack @line47 

A few thoughts:
Why is the contact page only accessible to unauthenticated users?  I ask because there  is mention of support.
Has there been consideration for contact page to be accessible through the navigation bar?

@line47 
Language suggestions:

- We provide support during U.S. East and West Coasts business hours. While we are unable **to** guarantee support outside of those hours, we’re always monitoring for issues that might affect customers. 
- I was initially going to suggest capitalizing cloug.gov everywhere but after looking at the https://cloud.gov/ it appears that cloud.gov is always lowercased ... even at the beginning of sentences.  Can you lowercase it on the contact page?

**Comment by annahsebok** [comment]
@amirbey thanks for catching. My thoughts:

- The contact page should be accessible to all users.
- I'd suggest we include a link to the contact page from the bottom nav (if this PR isn't doing that already...I thought it was).
- Thanks for catching those copy errors. I've pushed a commit to fix them!

**Comment by amirbey** [comment]
@annahsebok 
sure. np. i only saw this link accessible towards the bottom of the home page.

@line47  - can you please make this page accessible to all users by removing the /sites redirect for authenticated users and also adding the navbar addition as suggested by @annahsebok . Thanks!

**Comment by line47** [comment]
Thanks ya'll I'll get these updates in today 👍

**Comment by wslack** [comment]
@line47 or @amirbey - I made two little updates to check - otherwise, let's merge!

**Comment by amirbey** [comment]
@wslack - i have reviewed and approve your changes.

**Pull request merged**